_posts/2022-11-10-why-python-package-health-metrics-matter.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: single
+title: "Why Thinking About Metrics Should be the First Thing That Any New Organization, Program or Project Does"
+excerpt: "Collecting baseline data that aligns with the goals and outcomes of your project, program or organization is critical to do at the beginning. Here I briefly explain why Python package health metrics are so important to the long-term success of pyOpenSci."
+author: "Leah Wasser"
+permalink: /blog/why-python-package-health-metrics-matter
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - highlight
+  - python-packaging
+  - peer-review
+toc: true
+comments: true
+---
+
```

Comment:
Should there be a brief 1-2 sentence intro here with a link back to the previous post about "why open source package health matters to scientists"?

I actually like way it starts now with the assertive "This isn't my first rodeo" but as I understand it now you want this to be a series of posts and I totally didn't get that on the first read. So intro could be short and sweet, like "In [my last post](link) I talked about why OSS package health matters to scientists. Now I want to talk about why it matters for pyOpenSci..."

Reply:
I do see now you say "in a previous post" below, linking back to "why OSS scientists". Maybe I'm just a bad reader/listener, and obviously you have your own voice, but I might be a bit more assertive about linking the posts right at the top

Reply:
ok will look at that. yes this is a three part series. so i do want this to be clear. i originally wrote it as one ginormous and then realized it was way too long with too many focus points.